dev/tests/integration/framework/Magento/TestFramework/TestCase/AbstractController.php

```diff
@@ -15,6 +15,7 @@
 
 /**
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
I am not sure myself. https://travis-ci.org/dmanners/magento2/jobs/257261073 that was the failure. I wonder if PHPMD runs from a diff of classes changed?